### ISSUE TITLE
Баг: Перестаёт работать панель управления

### ISSUE THREAD

**Issue by Gelezako** [sergejey/majordomo#125]
**Шаги для воспроизведения:**
1. Зайти в панель управления - раздел "пользователи".
2. Добавить нового пользователя, не админа и не по умолчанию.
3. Закрыть все вкладки и подождать некоторое время.
4. Открыть приложение снова и нажать на кнопку "панель управления". (баг не проявляется сразу, у меня проявляется на следующий день, но скорей всего это происходит раньше, возможно после того как пропадет сессия.)

**Фактический результат:** Панель управления открыта не будет. Редирект на главную страницу.

**Нотс:** Если через phpmyadmin4 удалить из таблицы users этого пользователя, то панель управления снова начинает работать.

**Comment by sergejey** [comment]
Я попробовал воспроизвести через открытие новой вкладки в режиме инкогнито -- в таком случае сессия не подхватывается. Получилось воспроизвести только на половину -- создал пользователя, отключил и ему и администратору галочку "по-умолчанию", в новой сессии я действительно сразу не смог войти в админку, т.к. не был выбран пользователь с правами администратора. Я просто нажал на ссылку выбора пользователей на домашней странице и выбрал администратора. Пароль у меня не был установлен, так что пользователь сразу активировался и я смог спокойно зайти в панель управления.

**Comment by Gelezako** [comment]
>  Я просто нажал на ссылку выбора пользователей на домашней странице и выбрал администратора.

что это за ссылка, как на неё нажать?

**Comment by sergejey** [comment]
Клик на фотографию в правом верхнем углу -- это окно выбора текущего пользователя, там так же нужно кликнуть на фотографию нужного пользователя.

Если у пользователя нет прав администратора, то он не может зайти в панель управления.

**Comment by nick7zmail** [comment]
Может стоит скрывать кнопку админки в этом случае, чтоб не смущала?)

**Comment by Gelezako** [comment]
Думаю не стоит её скрывать, потому что мы должны иметь возможность выбора пользователя под которым хотим зайти.

**Comment by nick7zmail** [comment]
Гость - просто любой пользователь без имени, а зареганный - с именем, своим объектом, и к нему можно цеплять какие либо методы, на реакцию местоположения к примеру. А кнопку панели управления не совсем же убрать...просто скрыть её для неадминов, а когда выбираешь админа - отображать.

**Comment by Gelezako** [comment]
Да, я понимаю что у зарегистрированного есть свойства итд. Я имел ввиду чем он отличается в плане доступа к  панели инструментов? А на счёт скрытия понял идею, возможно так и лучше. Зачем отображать контрол если по нему нельзя кликнуть.

**Comment by nick7zmail** [comment]
В плане доступа к панели управления - скорее всего никак. Обоим запрещён. А вот в плане доступа к отдельным элементам сцен/меню - вполне могут отличаться. К примеру авторизованным (домашним) разрешать все элементы. Гостям, к примеру, только светом пощелкать, не более.

**Comment by Gelezako** [comment]
Хм, и каким образ такие права можно раздать? Это на уровне сцен или как ?

**Comment by nick7zmail** [comment]
![image](https://user-images.githubusercontent.com/15828541/37733078-b86e84a2-2d68-11e8-9d30-391781d6f055.png)
Пунктик есть во всех элементах сцен и элементах меню - контроль доступа...

**Comment by Aldaran** [comment]
Столкнулся с такой проблемой, установлен пароль на вход с нелокальной сети (EXT_ACCESS). И пароль на вход по паролю на администратора. Все =). Нормально захожу на главную страницу по паролю, но дальше не могу войти как пользователь-администратор, требует пароль, который не принимает. С локальной сети, спокойно захожу по паролю.

**Comment by andrey-melnyk** [comment]
то же, что у автопа, не могу теперь никак попасть в админку, отовсюде редиректит на главную

**Comment by andrey-melnyk** [comment]
короче да, надо ручками в базе выставить дефолтного юзера, тогда работает. Спасибо автору за тему, а то потратил кучу времени шоб найти причину....

**Comment by tarasfrompir** [comment]
> Пунктик есть во всех элементах сцен и элементах меню - контроль доступа... ![image](https://user-images.githubusercontent.com/15828541/37733078-b86e84a2-2d68-11e8-9d30-391781d6f055.png)

Ну практически не во всех, но втрети точно есть
